package.json

```diff
@@ -49,7 +49,7 @@
     "@geosolutions/acorn-jsx": "4.0.2",
     "@geosolutions/jsdoc": "3.4.4",
     "@geosolutions/mocha": "6.2.1-3",
-    "@mapstore/eslint-config-mapstore": "1.0.5",
+    "@mapstore/eslint-config-mapstore": "file:./utility/eslint",
```

Comment:
This should not be committed

Reply:
of course, i did it so the action-build could have used the new config, it's also written in the PR description when i say PR is not mergiable